src/Concordium/Client/Cli.hs

```diff
@@ -360,7 +360,25 @@ data ConsensusStatusResult = ConsensusStatusResult
   , csrFinalizationCount :: Int
   , csrLastFinalizedTime :: Maybe UTCTime
   , csrFinalizationPeriodEMA :: Maybe Double
-  , csrFinalizationPeriodEMSD :: Maybe Double }
+  , csrFinalizationPeriodEMSD :: Maybe Double
+  , csrProtocolVersion :: ProtocolVersion
+  , csrGenesisIndex :: GenesisIndex
+  , csrCurrentEraGenesisBlock :: !BlockHash
+  , csrCurrentEraGenesisTime  :: !UTCTime }
+
+-- The JSON fields protocolVersion, genesisIndex, currentEraGenesisBlock and currentEraGenesisTime
+-- are introduced in protocol version 2, so if these are not returned by the node, we assume¨
+-- that the node is running in protocol version 1. We therefore use the default values
+-- * P1 for protocolVersion 
+-- * 0 for genesisIndex since the first protocol update to increase the genesisIndex is
+--   assumed to change the protocol version to 2, and thus we can assume that the
+--   genesisIndex is 0, if it isn't returned by the node.
+-- * csrGenesisBlock for currentEraGenesisBlock since the first protocol update
+--   changes the protocol version to 2, meaning that if currentEraGenesisBlock
+--   is not returned by the node, then the current genesis block is the very
+--   first genesis block (since no regenesis would have happened yet)
+-- * csrGenesisTime for currentEraGenesisTime since (again) no regenesis 
+--   would have happened if currentEraGenesisTime is not returned by the node.
```

Comment:
We have discussed today that there is no need to keep this backwards compatibility so I recommend removing this default handling.

This will also make it easier to refactor in the future and use the ConsensusStatus type from base without it being a breaking change.